molgenis-dataexplorer/src/main/java/org/molgenis/dataexplorer/controller/DataExplorerController.java

```diff
@@ -323,32 +323,36 @@ else if (permissionService.hasPermissionOnEntityType(entityTypeId, Permission.CO
 		return modulesConfig;
 	}
 
-	@GetMapping("/packageHref")
+	@GetMapping("/navigatorLinks")
 	@ResponseBody
-	public LinkedList<Href> getPackageLink(@RequestParam("entity") String entityTypeId)
+	public LinkedList<NavigatorLink> getNavigatorLinks(@RequestParam("entity") String entityTypeId)
```

Comment:
Intelij and Sonar warn about using impl of List 'LinkedList' and suggest using 'List'